NodeOS.svg

```diff
@@ -3,6 +3,7 @@
    xmlns="http://www.w3.org/2000/svg"
    width="56cm"
    height="60cm"
+   shape-rendering="geometricPrecision"
```

Comment:
It outputs a really nice image compared to what I've got before. I'll test it with crispEdges again, but I'm not use if crispEdges really improves the quality of the picture